static/app/components/discover/performanceCardTable.tsx

```diff
@@ -0,0 +1,392 @@
+          {/* Table This Release column */}
+          {columnData()}
+        </div>
+        <div>
+          {/* Table Change column */}
+          {columnData()}
+        </div>
+      </Fragment>
+    );
+  }
+
+  renderUnknownPerformance() {
+    return (
+      <Fragment>
+        <div>
+          {/* Table description column */}
+          <PanelItem>{t('User Misery')}</PanelItem>
+        </div>
+        <div>
+          {/* TODO(kelly): placeholder data. will need to add user misery data in follow-up pr */}
+          {this.userMiseryField()}
+        </div>
+        <div>
+          {/* Table All Releases column */}
+          {this.userMiseryField()}
+        </div>
+        <div>
+          {/* Table This Release column */}
+          {this.userMiseryField()}
+        </div>
+      </Fragment>
+    );
+  }
+
+  render() {
+    const {project, organization, isLoading, isEmpty} = this.props;
+    // Custom set the height so we don't have layout shift when results are loaded.
+    const loader = <LoadingIndicator style={{margin: '70px auto'}} />;
+
+    const title = FRONTEND_PLATFORMS.includes(project.platform as string)
+      ? t('Frontend Performance')
+      : BACKEND_PLATFORMS.includes(project.platform as string)
+      ? t('Backend Performance')
+      : MOBILE_PLATFORMS.includes(project.platform as string)
+      ? t('Mobile Performance')
+      : t('[Unknown] Performance');
+    const platformPerformance = FRONTEND_PLATFORMS.includes(project.platform as string)
+      ? this.renderFrontendPerformance()
+      : BACKEND_PLATFORMS.includes(project.platform as string)
+      ? this.renderBackendPerformance()
+      : MOBILE_PLATFORMS.includes(project.platform as string)
+      ? this.renderMobilePerformance()
+      : this.renderUnknownPerformance();
+
+    return (
+      <Fragment>
+        <HeadCellContainer>{title}</HeadCellContainer>
+        {title.includes(t('Unknown')) ? (
+          <StyledAlert type="warning" icon={<IconWarning size="md" />} system>
+            For more performance metrics, specify which platform this project is using in{' '}
+            <Link to={`/settings/${organization.slug}/projects/${project.slug}/`}>
+              project settings.
+            </Link>
+          </StyledAlert>
+        ) : null}
+        <StyledPanelTable
+          isLoading={isLoading}
+          isEmpty={isEmpty}
+          emptyMessage={t('No transactions found')}
+          headers={[
+            <Cell key="description" align="left">
+              {t('Description')}
+            </Cell>,
+            <Cell key="releases" align="right">
+              {t('All Releases')}
+            </Cell>,
+            <Cell key="release" align="right">
+              {t('This Release')}
+            </Cell>,
+            <Cell key="change" align="right">
+              {t('Change')}
+            </Cell>,
+          ]}
+          disablePadding
+          loader={loader}
+          disableTopBorder={title.includes('Unknown')}
```

Comment:
We should not do these checks against the title (same on line 295) - it can get wonky with translations and can break once we change some copy. How about we create a boolean called `isUnknownPlatform`, it could look like this:
```
const isUnknownPlatform = ![...FRONTEND_PLATFORMS, ...BACKEND_PLATFORMS, ...MOBILE_PLATFORMS ].includes(project.platform)
```